ios/swift/AmpliSwiftSampleApp/README.md

```diff
@@ -0,0 +1,33 @@
+# Ampli iOS SDK (Swift)
+An example app using the Ampli Swift SDK
+
+# Usage
+
+### Setup the project
+You will need to do the following before running the app.
+1.Set up environment variable with your API keys
+   1. Go to schema edit and find `Environment Variables` section
```

Comment:
```suggestion
   1. Go to `Edit Scheme > Run > Arguments > Environment Variables` section
```

Can you please provide a little more detail here. Ideally a person who has no idea how to use XCode would be able to get this up a running without additional information.

Reply:
Sure, let me add the suggested changes.

Reply:
updated